src/service/createUpdateIdentifierPersona.ts

```diff
@@ -0,0 +1,112 @@
+import NoModel from 'jscommons/dist/errors/NoModel';
+import * as promiseRetry from 'promise-retry';
+import Locked from '../errors/Locked';
+import CreateUpdateIdentifierPersonaOptions // tslint:disable-line:import-spacing
```

Comment:
Did the linter not autofix this to some spacing it would accept?

Reply:
No, then it goes over line length.